telecommunicationCableColorKit.cpp

```diff
@@ -0,0 +1,44 @@
+#include "cableColorPlugins.h"
+#include <assert.h>
+#include <iostream>
+#include <vector>
+#include <map>
+
+void testNumberToColorCablePair(int colorCablePairNumber, TelecommunicationCableColorCoder::MajorColorCables expectedMajorColorCable, TelecommunicationCableColorCoder::MinorColorCables expectedMinorColorCable, std::map<std::string, int>& manual){
```

Comment:
This test code itself prints the manual? What if the formatting in the manual needs to be tested?

Reply:
sorry sir i did not get this, could you please describe little bit for my understanding.